### ISSUE TITLE
Fix hardcode in gulpfile.js

### ISSUE THREAD

**Issue by dvc** [shower/shower#274]
https://github.com/shower/shower/blob/master/gulpfile.js#L25
` /(<link rel="stylesheet" href=")(node_modules\/shower-ribbon\/)(styles\/screen-16x10.css">)/g,`

I can\`t replace 
`<link rel="stylesheet" href="node_modules/shower-ribbon/styles/screen-16x10.css">`
to
`<link rel="stylesheet" href="node_modules/shower-ribbon/styles/screen-4x3.css">`

according to documentation:
https://github.com/shower/shower/blob/master/docs/features-en.md#slide

>  To enable needed one include appropriate style file screen-4x3.css or screen-16x10.css

It`s a bug

**Comment by dvc** [comment]
More universal rule:

```
		.pipe(replace(
			/(<link rel="stylesheet" href=")(node_modules\/shower-)([^\/]*)\/(.*\.css">)/g,
			'$1shower/themes/$3/$4', { skipBinary: true }
		))

```

**Comment by pepelsbey** [comment]
Oh, that’s a bug, thank you for spotting this. But I wouldn’t make it that greedy it could match `shower-other/styles/subfolder/any_named.css`. I’d adjust it for any theme name and any ratio postfix. I’ll fix it tomorrow.

BTW, backtick is the worst symbol for imitating an apostrophe. It should be either `’` or `'` if you don’t want to bother, but definitely not backtick.

backtick <img width="212" alt="screen shot 2016-06-29 at 23 55 33" src="https://cloud.githubusercontent.com/assets/105274/16468550/1c643ae2-3e55-11e6-96f7-2f6d14f7743f.png">
apostrophe <img width="213" alt="screen shot 2016-06-29 at 23 55 44" src="https://cloud.githubusercontent.com/assets/105274/16468554/1f8931c8-3e55-11e6-8a89-dbe00f746f54.png">
single quote <img width="220" alt="screen shot 2016-06-29 at 23 55 56" src="https://cloud.githubusercontent.com/assets/105274/16468557/2472c294-3e55-11e6-8720-7fad9a52777e.png">

**Comment by dvc** [comment]
Thanks, i'll remember it ;)

**Comment by pepelsbey** [comment]
Fixed in ed650220 with exact regexp you suggested. Sorry it took so long and thank you!
